pkg/reloader/reloader.go

```diff
@@ -363,11 +364,13 @@ func (r *Reloader) apply(ctx context.Context) error {
 		}
 		r.reloads.Inc()
 		if err := r.triggerReload(ctx); err != nil {
+			r.forceReload = true
```

Comment:
Even if we set it to true, this condition is not used during the retry process. Why we don't set it on line 385?

Reply:
you are right, I have moved it to line 385 :)